client/result/result.go

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Sold! Bits shared between `repair` and `bootstrap` -> `client`. Rest of `client/result` -> `bootstrap/result`.